<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="move_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>移动窗口</string>
    </property>
   </widget>
   <widget class="QPushButton" name="get_pose">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>获取位置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="change">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>改位置大小</string>
    </property>
   </widget>
   <widget class="QPushButton" name="modaldlg">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>显示模态对话框</string>
    </property>
   </widget>
   <widget class="QPushButton" name="msgbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>msg_box</string>
    </property>
   </widget>
   <widget class="QPushButton" name="file_dialog">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>File_Dialog</string>
    </property>
   </widget>
   <widget class="QPushButton" name="font_dlg">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>font_dlg</string>
    </property>
   </widget>
   <widget class="QLabel" name="fontlabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>你好</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="color_group">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>291</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>color_group</string>
    </property>
    <widget class="QPushButton" name="color_dlg">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>color_dlg</string>
     </property>
    </widget>
    <widget class="QLabel" name="color_label">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="input_dialog">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>100</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>input_dialog</string>
    </property>
   </widget>
   <widget class="QPushButton" name="progress_dlg">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>140</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>progress_dlg</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menutext">
    <property name="title">
     <string>text</string>
    </property>
    <addaction name="open_action"/>
    <addaction name="separator"/>
    <addaction name="save_action"/>
   </widget>
   <addaction name="menutext"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="save_action"/>
   <addaction name="separator"/>
   <addaction name="open_action"/>
  </widget>
  <action name="open_action">
   <property name="text">
    <string>open_text</string>
   </property>
  </action>
  <action name="save_action">
   <property name="text">
    <string>save_text</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
